AudioKitSynthOne/DSP/Kernel/S1DSPKernel.hpp

```diff
@@ -220,7 +220,18 @@ class S1DSPKernel : public AKSoundpipeKernel, public AKOutputBuffered {
     
     float monoFrequencySmooth = 261.6255653006f;
 
+    // S1TuningTable protocol
+    void setTuningTable(float value, int index);
+    float getTuningTable(int index);
+    void setTuningTableNPO(int npo);
+
 private:
+    std::array<std::atomic<float>, 128> tuningTable;
```

Comment:
I wonder if we should default initialize the `std array` somewhere just for safety, maybe in the init function by calling `tuningTable.fill(0);` or whatever number makes sense?

Reply:
1 is a reasonable default

Reply:
1 is a reasonable default

Reply:
done